docs/velocity/dev/getting-started/api-basics.mdx

```diff
@@ -93,13 +93,17 @@ Dependencies on other plugins are also to be specified there, but we'll get to t
 
 ### A word of caution
 
+:::caution
+
 In Velocity, plugin loading is split into two steps: construction and initialization. The code in
 your plugin's constructor is part of the construction phase. There is very little you can do safely
 during construction, especially as the API does not specify which operations are safe to run during
 construction. Notably, you can't register an event listener in your constructor, because you need to
 have a valid plugin registration, but Velocity can't register the plugin until the plugin has been
 constructed, causing a "chicken or the egg" problem.
 
+:::
+
 To break this vicious cycle, you should always wait for initialization, which is indicated when
```

Comment:
I don't like this; this text is now a continuation of the admonition's content, but admonitions are supposed to be self-contained pieces of advice that don't interrupt regular document flow

Reply:
Makes sense